### DISCUSSION TITLE
有没有办法禁止掉网页端进入视频后的自动播放和自动连播

### DISCUSSION THREAD

**Discussion by sssstar** [the1812/Bilibili-Evolved#3457] [功能建议 / Ideas]
如果视频有观看进度，再次进入后就会直接跳到进度自动播放，十分难受。
自动连播也是，手动关了，不知道为啥它还会自己打开。

**Comment by Geeyun-JY3** [comment]
> 如果视频有观看进度，再次进入后就会直接跳到进度自动播放，十分难受。

#1191

自动连播的话你试试 [传统连播模式](https://github.com/the1812/Bilibili-Evolved/blob/master/doc/features/features.md#传统连播模式) 吧

**Reply by sssstar** [reply]
破案了兄弟，我把传统连播模式关掉，就都好了。
